tests/dualtor/test_tor_ecn.py

```diff
@@ -103,8 +104,7 @@ def build_encapsulated_ip_packet(
 
     inner_ttl = random.choice(list(range(3, 65)))
     inner_ecn = random.choice(list(range(0, 3)))
-    if is_tunnel_qos_remap_enabled(tor):
```

Comment:
Why remove the check `is_tunnel_qos_remap_enabled` here?

Reply:
This condition check is redundant. We already have the same check in derive_out_dscp_from_inner_dscp(). And because of this redundant check when is_tunnel_qos_remap_enabled() returns false we have an undefined outer_dscp which has to be used further in the code flow.

{noformat}
def derive_out_dscp_from_inner_dscp(duthost, inner_dscp):
    """
    Helper function to find outer DSCP ID for a inner DSCP ID.
    """
    if is_tunnel_qos_remap_enabled(duthost):
        tc_to_dscp_map_name = "AZURE_TUNNEL"
        map = dut_dscp_tc_queue_maps(duthost)
        # Load tc_to_dscp_map
        dscp_id = map['tc_to_dscp_map'][tc_to_dscp_map_name][str(inner_dscp)]
        return int(dscp_id)
    else:
        return inner_dscp
{noformat}